packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/__tests__/quanticFeedbackModal.test.js

```diff
@@ -0,0 +1,436 @@
+// @ts-ignore
+import {createElement} from 'lwc';
+import QuanticFeedbackModal from '../quanticFeedbackModal';
+
+const functionsMocks = {
+  submitFeedback: jest.fn(),
+};
+
+const selectors = {
+  errorMessage: '.error-message',
+  submitButton: '.feedback-modal-footer__submit',
+  successMesage: '.feedback-modal-body__success-message',
+  radioGroup: 'lightning-radio-group',
+  textarea: 'textarea',
+  detailsErrorMessage: '.feedback-modal-body__details-error-message',
+};
+
+const exampleValue = 'example value';
+const exampleLabel = 'example label';
+const exampleDetails = 'example details';
+/** @type {Array} */
+const exampleOptions = [{label: exampleLabel, value: exampleValue}];
+
+const errors = {
+  invalidArrayError: 'The options provided are not in a valid array.',
+  emptyArrayError: 'At least one option must be specified.',
+  missingLabelOrValueError:
+    'Each option requires a label and a value to be specified.',
+  missingSubmitFeedback: 'The submitFeedback property is not a valid function.',
+  invalidLabelTypeError: (label) =>
+    `The "${label}" label is not a valid string.`,
+  invalidValueTypeError: (value) =>
+    `The "${value}" value is not a valid string.`,
+};
+
+const defaultOptions = {
+  options: exampleOptions,
+  submitFeedback: functionsMocks.submitFeedback,
+};
+
+function createTestComponent(options = defaultOptions) {
+  const element = createElement('c-quantic-feedback-modal', {
+    is: QuanticFeedbackModal,
+  });
+
+  for (const [key, value] of Object.entries(options)) {
+    element[key] = value;
+  }
```

Comment:
it would do the same thing, but personally I feel like the way we currently using looks like a more natural and readable way to affect props to the component.